Real-Time Corruptor/BizHawk_RTC/StandaloneRTC/Program.cs

```diff
@@ -18,12 +17,14 @@ static void Main(string[] args)
 		{
 
 			//in case assembly resolution fails, such as if we moved them into the dll subdiretory, this event handler can reroute to them
-			AppDomain.CurrentDomain.AssemblyResolve += new ResolveEventHandler(CurrentDomain_AssemblyResolve);
+			AppDomain.CurrentDomain.AssemblyResolve += CurrentDomain_AssemblyResolve;
 
 			var processes = Process.GetProcesses().Select(it => $"{it.ProcessName.ToUpper()}").OrderBy(x => x).ToArray();
 
             int nbInstances = 0;
-            foreach (var prc in processes)
+			//this could be replaced with linq expression
+			//I replaced this with explicit type because it should always be a string
+            foreach (string prc in processes)
```

Comment:
If you want to change it to a linq expression, go ahead. Otherwise just remove those comments

Reply:
Changed to LINQ  removed comment